worker/provisioner/export_test.go

```diff
@@ -38,10 +38,8 @@ func GetRetryWatcher(p Provisioner) (watcher.NotifyWatcher, error) {
 var (
 	GetContainerInitialiser = &getContainerInitialiser
 	GetToolsFinder          = &getToolsFinder
-	ResolvConfFiles         = &resolvConfFiles
```

Comment:
ResolvConfFiles and CombinedCloudInitData only used by the container broker tests, moved.